src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -30,8 +32,14 @@ public static double runMean(){
         return 0;
     }
 
-    public static double runVariance(){
-        return 0;
+    public void runVariance(){
+        System.out.println("Run task Rainfall: variance method");
+        System.out.println("Input the name of the city");
+        String town = Reader.readString();
+        System.out.println("Input string with rainfall records");
```

Comment:
Mb rules should be specified here?
From Kata : "The records of towns are separated by \n. The name of each town is followed by :"

Reply:
If we wouldn't show here task description, we should specify rules of such input - records of towns should be separated by '\n' and to separate town name and data use ':'